src/main/java/com/epam/ta/reportportal/dao/TestItemRepositoryCustomImpl.java

```diff
@@ -389,6 +388,40 @@ public TestItemTypeEnum getTypeByItemId(Long itemId) {
 		return PageableExecutionUtils.getPage(items, pageable, () -> dsl.fetchCount(QueryBuilder.newBuilder(filter).build()));
 	}
 
+	@Override
+	public List<NestedStep> findAllNestedStepsByIds(Collection<Long> ids) {
+
+		Field<Long> nestedParentIdField = fieldName(NESTED, TEST_ITEM.PARENT_ID.getName()).cast(Long.class);
```

Comment:
I meant 
Table<...> nested = TEST_ITEM.as(NESTED)
and then use nested.field in the query

Reply:
fixed